frontend/src/components/Modal/ConfirmationModal.tsx

```diff
@@ -24,26 +32,56 @@ const ConfirmationModal = ({ open, rider, onClose }: ConfirmationProps) => {
     onClose();
   };
 
-  const studentDelete = () => {
+  // userType should be either 'admin', 'driver', 'rider'
+  const deleteAPICall = (
+    userType: string,
+    userId: string,
+    refreshFunc: () => Promise<void>
+  ) => {
+    const userGroup = `${userType}s`;
     fetch(
-      `/api/riders/${!rider ? '' : rider.id}`,
+      `/api/${userGroup}/${userId ? userId : ''}`,
       withDefaults({
         method: 'DELETE',
       })
     )
-      .then(refreshRiders)
+      .then(refreshFunc)
       .then(() => {
-        history.push('/riders');
-        showToast('The student has been deleted.', ToastStatus.SUCCESS);
+        history.push(
```

Comment:
I think `/admin` is already present in the path, and we only need to redirect to `riders` or `employees` page.
This seems to work perfectly:
history.push(`/${userType === 'rider' ? 'riders' : 'employees'}`);